Makefile

```diff
@@ -32,8 +32,10 @@ BUILDTAGS ?= \
 	exclude_graphdriver_devicemapper \
 	seccomp \
 	varlink
-PYTHON ?= $(shell command -v python python3|head -n1)
+PYTHON ?= $(shell command -v python3 python|head -n1)
 PKG_MANAGER ?= $(shell command -v dnf yum|head -n1)
+# ~/.local/bin is not in PATH on all systems
+PRE_COMMIT ?= $(shell command -v ~/.local/bin/pre-commit pre-commit | head -n1)
```

Comment:
Assuming I understand `pip` correctly (which is not certain, because I'm not a Python fan), this should probably be `$(PYTHON) -m site --user-base` instead of hardcoded `~/.local`